environment.yml

```diff
@@ -20,6 +20,9 @@ dependencies:
   - qgis ~=3.16.0
   - requests ~=2.25.0
   - types-requests ~=2.25.0
+  - netCDF4 ~=1.5
+  - pandas ~=1.3
+  - openpyxl ~= 3.0
```

Comment:
Since this isn't an "imported" dependency, can we move it to a different block or create a new block for these types of dependencies? Not "imported" but it's like a "plugin" to something we _do_ import? :shrug: Not sure how to describe that category.